test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,8 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+  test("returns array of human, cat and dog years when passed human years", () => {
+    expect(humanCatDogYears(10)).toStrictEqual([10, 56, 64]);
+    expect(humanCatDogYears(6)).toStrictEqual([6, 40, 44]);
```

Comment:
You want your tests to test every possible outcome of your solution, so you should also put in a test case of human years of 1 and 2 to make sure you're testing what happens when either of your other if cases are true.

Reply:
Add those extra test cases in and see if your function still passes.